aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  Main,
+  Text,
+  TextInput,
+} from 'grommet';
+import { Close, Next } from 'grommet-icons';
+
+import { FormContainer } from '.';
+import { emailValidation } from './formHelpers';
+
+const ResetPassword = ({ closeLayer, email }) => {
+  const [formValues, setFormValues] = React.useState({ resetEmail: email });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your password reset logic here
+    // Display success status
+    closeLayer();
+  };
+
+  return (
+    <>
+      <Box
+        direction="row"
+        justify="end"
+        pad={{ horizontal: 'small', top: 'small' }}
+      >
+        <Button
+          a11yTitle="Close reset password form"
+          icon={<Close />}
+          onClick={closeLayer}
+        />
+      </Box>
+      <Box
+        gap="medium"
+        margin={{ horizontal: 'xlarge', bottom: 'xlarge', top: 'large' }}
+        width="medium"
+      >
+        <Heading level={2} margin="none">
+          Reset Password
+        </Heading>
+        <Form
+          validate="blur"
+          value={formValues}
+          onChange={setFormValues}
+          onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+        >
+          <Box gap="medium">
+            <Text>
+              An email to reset your password will be sent to the following
+              address:
+            </Text>
+            <FormField
+              name="resetEmail"
+              component={TextInput}
```

Comment:
I think we should be guiding folks to not use `component`. Instead:
```
<FormField name="resetEmail" ...>
  <TextInput name="resetEmail" .../>
</FormField>
```

Reply:
Noted. Wasn't sure the direction for which we wanted to advocate. Will update throughout.

Reply:
@ericsoderberghp Implemented child approach rather than `component`